litmus-portal/graphql-server/pkg/authorization/user_jwt.go

```diff
@@ -36,3 +36,25 @@ func UserValidateJWT(token string) (jwt.MapClaims, error) {
 
 	return nil, errors.New("Invalid Token")
 }
+
+func GetUsername(token string) (string, error) {
+	tkn, err := jwt.Parse(token, func(token *jwt.Token) (interface{}, error) {
```

Comment:
Here all these validations are unnecessary as the token has already been validated previously.